core/controllers/topics_and_skills_dashboard_test.py

```diff
@@ -286,7 +286,16 @@ def test_fetch_filtered_skills_with_given_cursor(self):
                 'num_skills_to_fetch': 1,
             }, csrf_token=csrf_token)
 
+        # Default sort is newly created first, and hence the following order
+        # because skill with skill_id is created last, then the skill with
+        # subtopic_skill_id and then the skill with linked_skill_id is created
+        # first.
```

Comment:
Can you rewrite this docstring in such a way that you don't use "created first" to mean two different things? It requires a careful reading to realize that, in the first case, you really mean "created last".

Reply:
Updated these docstrings.